gazebo_ros2_control_demos/config/cartpole_controller.yaml

```diff
@@ -0,0 +1,17 @@
+cart_pole_controller:
```

Comment:
When I try

`ros2 launch gazebo_ros2_control_demos cart_example_position_pid.launch.py`

I get

```
[gzserver-1] [ERROR] [1592959789.018932053] [ros2_control_plugin]: Loading controller_manager
[gzserver-1] [INFO] [1592959789.030798125] [gazebo_controller_manager]: Loading controller 'cart_pole_controller'
[gzserver-1] 
[gzserver-1] [ERROR] [1592959789.031255797] [gazebo_controller_manager]: Loader for controller 'cart_pole_controller' not found

```

Reply:
@ahcorde , I saw you pushed some commits since this comment, but I'm still having this problem.

Reply:
If I understand correctly, this PR depends on #25 to work. In that case, I think it should be rebased on top of that, otherwise there's no way to test it.